client/src/components/CategoryExpenses.tsx

```diff
@@ -11,123 +12,193 @@ interface IProps {
 	eSetFilter: (term: string, match: string) => void
 }
 
-export class CategoryExpenses extends React.Component<IProps, {}> {
-	constructor(props: IProps) {
-		super(props)
+// @ts-ignore
```

Comment:
don't ts-ignore this, it's giving you a valid ts error about why the code is invalid ts

Reply:
remove the `: React.FC<IProps>` and define the props you're passing in with `: IProps`